src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDescriptor.cs

```diff
@@ -147,5 +151,11 @@ public class TagHelperDescriptor
         /// The <see cref="TagHelperUsageDescriptor"/> that indicates how to use the tag helper at design time.
         /// </summary>
         public TagHelperUsageDescriptor UsageDescriptor { get; }
+
+        /// <summary>
+        /// The HTML element the tag helper may output.
+        /// </summary>
+        /// <remarks>Overrides the HTML IntelliSense provided at design time.</remarks>
```

Comment:
This is a Visual Studio and perhaps OmniSharp-specific comment and should not be so definite. Suggest "In IDEs supporting Intellisense, may override the HTML information provided at design time."